calculator.sh

```
@@ -6,7 +6,11 @@ echo "Enter first number:"
 read num1
 
 echo "Enter second number:"
-read num4
+<<<<<<< HEAD
```

Shellcheck (0.10.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)